integration-tests/src/test/java/oracle/weblogic/kubernetes/ItFmwDiiSample.java

```diff
@@ -45,9 +52,10 @@
 import static oracle.weblogic.kubernetes.assertions.TestAssertions.domainExists;
 import static oracle.weblogic.kubernetes.utils.ApplicationUtils.callWebAppAndWaitTillReady;
 import static oracle.weblogic.kubernetes.utils.CommonTestUtils.checkPodReadyAndServiceExists;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.getActualLocationIfNeeded;
 import static oracle.weblogic.kubernetes.utils.CommonTestUtils.getNextFreePort;
 import static oracle.weblogic.kubernetes.utils.CommonTestUtils.testUntil;
-import static oracle.weblogic.kubernetes.utils.CommonTestUtils.withLongRetryPolicy;
+//import static oracle.weblogic.kubernetes.utils.CommonTestUtils.withLongRetryPolicy;
```

Comment:
remove commented code

Reply:
Done.

Reply:
Pani, this is the doc: https://oracle.github.io/weblogic-kubernetes-operator/samples/domains/domain-home-in-image/. As you can see "Configuration parameters" section, wdtVersion and witVersion are configurable parameters for the customer, which I didn't change in this PR (per our discussion). So I don't think I need to change change any public documentation.